src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -198,12 +199,40 @@ private CallbackValidations() {
             .map(c -> (String) c);
     }
 
+    private static Optional<String> getAwaitingPaymentDcnProcessing(CaseDetails theCase) {
+        return Optional.ofNullable(theCase)
+            .map(CaseDetails::getData)
+            .map(data -> data.get("awaitingPaymentDCNProcessing"))
+            .map(c -> (String) c);
+    }
+
     static boolean hasOcr(CaseDetails theCase) {
         return getOcrData(theCase)
             .map(CollectionUtils::isNotEmpty)
             .orElse(false);
     }
 
+    static Validation<String, Void> validatePayments(
+        CaseDetails theCase,
+        Classification classification,
+        ServiceConfigItem config
+    ) {
+        Optional<String> awaitingPaymentsOptional = getAwaitingPaymentDcnProcessing(theCase);
+
+        if (!awaitingPaymentsOptional.isPresent()
+            || (awaitingPaymentsOptional.get().equals("No") // no payments or no payments pending for completion
+                || (awaitingPaymentsOptional.get().equals("Yes") // payments processing pending
```

Comment:
by any chance, do we have string constants for ccd yes/no?

Reply:
yes, updated